tests/letstest/scripts/test_letsencrypt_auto_certonly_standalone.sh

```diff
@@ -17,27 +17,6 @@ letsencrypt-auto certonly --no-self-upgrade -v --standalone --debug \
                    --register-unsafely-without-email \
                    --domain $PUBLIC_HOSTNAME --server $BOULDER_URL
 
-# we have to jump through some hoops to cope with relative paths in renewal
```

Comment:
It looks like the permissions on this file were changed and the execute permission was lost. Can we add it back?

Reply:
Corrected. As a side note: I would not trust the GIT client to always setup the executable permissions for me, as this kind of modification are quite hard to see. You should add a `chmod +x` at the relevant place for letstest/scripts/*.sh during the setup of letstest.

Reply:
Sounds good! Want to make an issue/PR?